content/_data/authors/krisstern.adoc

```diff
@@ -6,4 +6,4 @@ linkedin: kris-stern
 ---
 
 // descriptive text comes here
-Kris has been helping out with Jenkins's GSoC participation organization since 2022 and has volunteered to be a project mentor. She has participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021. Professionally Kris works in the IT sector as a software engineer / programmer. She has work experience in Python, C++, Java, JavaScript, HTML, CSS/Sass, JQuery, SQL, PHP, and have completed projects in software development, artificial intelligence/deep learning/computer vision, Qt programming, and web development. Kris is passionate about open-source and would like to share this passion with fellow learners.
\ No newline at end of file
+Kris has been helping out with Jenkins's GSoC participation organization since 2022 and has volunteered to be a GSoC project mentor. She has participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021. Professionally Kris works in the IT sector as a software engineer. She has work experiences in Python, C++, Java, JavaScript, TypeScript, HTML, CSS/Sass, JQuery, SQL, and have completed projects in software development in general, specifically in artificial intelligence/deep learning/computer vision, Qt programming, and web development. Kris is passionate about open-source and would like to share this passion with fellow learners. Currently, Kris is a part-time MCIT Online student at UPenn.
```

Comment:
```suggestion
Kris has been helping out with Jenkins' GSoC participation organization since 2022 and has volunteered to be a GSoC project mentor. She has participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021. Professionally, Kris works in the IT sector as a software engineer. She has work experiences in Python, C++, Java, JavaScript, TypeScript, HTML, CSS/Sass, JQuery, SQL, and has completed projects in software development in general and specifically in artificial intelligence/deep learning/computer vision, Qt programming, and web development. Kris is passionate about open-source and would like to share this passion with fellow learners. Currently, Kris is a part-time MCIT Online student at UPenn.
```
Suggestion to adjust some of the grammar and wording for clarity.